Makefile

```diff
@@ -138,3 +138,10 @@ lint-scripts:
 .PHONY: lint-styles
 lint-styles:
 	cd assets && npx stylelint $(STYLES_PATTERN)
+
+# Static code analysis
+# ----------------------------
+
+.PHONY: static-analysis
+static-analysis: ## Run static code analysis
```

Comment:
Should we just create a new `check-` target? This is how most of the other tools are used.

Something like `make check-static-analysis`.

Reply:
Yes, I found `check` to be a synonym to `analisys`, but consistency always wins!

`check-static-typing` might be better?